framework/src/window.cpp

```diff
@@ -454,7 +454,11 @@ namespace gvk
 		fill_in_present_semaphore_dependencies_for_frame(renderFinishedSemaphores, renderFinishedSemaphoreStages, current_frame());
 
 		if (!has_consumed_current_image_available_semaphore()) {
-			LOG_WARNING(fmt::format("Frame #{}: User has not consumed the 'image available semaphore'. Render results might be corrupted. Use consume_current_image_available_semaphore() every frame!", current_frame()));
+			
+			// TODO: this is not a great way of handling the spam. find a better solution (improved logger mechanism?)
+			// This message is important, however not necessary to output every frame when invokee is just disabled.
+			if (current_frame() % 5000 == 0)
+				LOG_WARNING(fmt::format("Timed message: The main invokee is disabled or user has not consumed the 'image available semaphore'. Render results might be corrupted. Use consume_current_image_available_semaphore() every frame!"));
```

Comment:
Please revert to the original code where the warning is issued every frame!
This is actually an error... don't remember exactly why I labeled it only as a warning.
I.e. this _should_ be displayed every frame.

There are some still unimplemented issues that could improve this, however. Also sync-refactoring might have a positive effect on this. It is definitely not something that should be fixed or altered in the context of Issue #65.

Reply:
I will change it back, but please be aware of the following situation:

This is not necessarily a bug or a mistake if invokees are disabled, which is a valid action. In that case nobody will consume the semaphore, and the resulting warning spam can be very distracting on 1000 frames per second, and disallow any background code that may be running to run properly (string operations = heavy).

In general, if there is any issue, I think it should be handled differently because the framework itself disallows render calls of disabled invokees, which in turn in my opinion is a logical inconsistency.

Reply:
This was resolved with a more general solution, as discussed.